docs/4.0/extend/icons.md

```diff
@@ -10,6 +10,7 @@ Bootstrap doesn't include an icon library by default, but we have a handful of r
 ## Preferred
 We've tested and used these icon sets ourselves.
 
+- [Feather](https://feathericons.com)
```

Comment:
Please, remove Feather link (http://colebemis.com/feather/) from the More options section below. Now you have a duplicate.